integration-cli/docker_cli_events_test.go

```diff
@@ -779,7 +779,7 @@ func (s *DockerSuite) TestEventsTop(c *check.C) {
 
 // #13753
 func (s *DockerSuite) TestEventsDefaultEmpty(c *check.C) {
-	dockerCmd(c, "run", "-d", "busybox")
+	dockerCmd(c, "run", "-d", "busybox", "top")
```

Comment:
@duglin  I'm agree with you, the test is to make sure `docker events` will not report events that already happened by default. so we should remove `-d` and `top`. Thanks, I'll updated